Purposal for Change: change the way to query the entrypoints

futbolsalas15 opened the pull request:
With this PR, I purpose the way to accessing the entrypoints to generate the d.ts files, because sometimes accessing through the chunks and querying issuer, sometimes has a null value, and didn't allow the generation of d.ts file.

unimonkiez commented:
I like it!  
Could you change back the strings to single quote though? I like it better that way.  
After that I will merge and publish your change

futbolsalas15 commented:
Hi!
I changed the double quotes to single quotes. And I don't know if it is right to you, but I added a .eslintrc.json file, so in future commits apply the rule of change double to single quotes (using eslinter).

Thanks again!

futbolsalas15 commented:
Hi @unimonkiez ,

What do you think of the latest update of my branch? Do you have more observations about that?

Thanks!

unimonkiez commented:
Awesome!
Sorry for not being responsive, school and stuff.
Now available on `v1.2.0`!
